test/base.js

```diff
@@ -14,6 +14,8 @@ var Promise = require('pinkie-promise');
 var yeoman = require('yeoman-environment');
 var userHome = require('user-home');
 
+var LF = process.platform === 'win32' ? '\r\n' : '\n';
```

Comment:
Same comment. Can we apply this fix and see if it makes Travis a little more happy?